src/components/AlertMessage/AlertMessage.tsx

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import styled from "styled-components";
+import { ThemeProvider } from "styled-components";
+import { CustomColors } from "../../colors/Colors";
+
+type DivMessageProps = {
+  isOpen: boolean;
+};
+
+const WarningDivMessage = styled.div<DivMessageProps>`
+  position: relative;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  border-color: ${(props) => props.theme.colors.softYellow};
+  color: ${(props) => props.theme.colors.olive};
+  background-color: ${(props) => props.theme.colors.lightYellow};
+  opacity: ${(props) => (props.isOpen ? 1 : 0)};
+  transition: all 0.2s;
+  visibility: ${(props) => (props.isOpen ? "visible" : "hidden")};
+`;
+
+const ErrorDivMessage = styled.div<DivMessageProps>`
+  position: relative;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  border-color: ${(props) => props.theme.colors.softRed};
+  color: ${(props) => props.theme.colors.burgundy};
+  background-color: ${(props) => props.theme.colors.lightRed};
+  opacity: ${(props) => (props.isOpen ? 1 : 0)};
+  transition: all 0.2s;
+  visibility: ${(props) => (props.isOpen ? "visible" : "hidden")};
+`;
+
+const SuccessDivMessage = styled.div<DivMessageProps>`
+  position: relative;
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  border-radius: 0.25rem;
+  border-color: ${(props) => props.theme.colors.softGreen};
+  color: ${(props) => props.theme.colors.darkGreen};
+  background-color: ${(props) => props.theme.colors.lightGreen};
+  opacity: ${(props) => (props.isOpen ? 1 : 0)};
+  transition: all 0.2s;
+  visibility: ${(props) => (props.isOpen ? "visible" : "hidden")};
+`;
+
+export const AlertMessage = ({
+  responseStatus,
+  isOpen,
+}: {
+  responseStatus: number;
+  isOpen: boolean;
+}) => {
+  <ThemeProvider theme={CustomColors}></ThemeProvider>;
+
+  switch (responseStatus) {
```

Comment:
tahle komponenta ve switch casu obsahuje hodně duplicitního kódu, nechceš to nějak zjednodušit???